### PR TITLE
CEDS-2469 - Change the Save and Continue button to Continue

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: app/controllers/declaration/ConsignmentReferencesController.scala [hunk 1]
```diff
@@ -21,8 +21,9 @@ import controllers.navigation.Navigator
 import forms.declaration.ConsignmentReferences
 import forms.declaration.ConsignmentReferences.form
 import javax.inject.Inject
+import models.DeclarationStatus.DeclarationStatus
 import models.requests.JourneyRequest
-import models.{DeclarationType, ExportsDeclaration, Mode}
+import models.{DeclarationStatus, DeclarationType, ExportsDeclaration, Mode}
```
Comment (PatrykRudnicki) [inline app/controllers/declaration/ConsignmentReferencesController.scala:26]: Those imports are not necessary after removing implementation from this file

Comment (SteveSugden) [reply app/controllers/declaration/ConsignmentReferencesController.scala:26]: I know - forgot to tidy imports in my rush to update the PR.   Next commit!

File: test/views/declaration/additionaldeclarationtype/DeclarationTypeViewSpec.scala [hunk 2]
```diff
@@ -115,69 +115,64 @@ class DeclarationTypeViewSpec extends UnitViewSpec with ExportsTestData with Com
...
-        view.getElementById("submit").text() mustBe messages(saveAndContinueCaption)
+        view.getElementById("submit").text() mustBe messages(continueCaption)
       }
 
       "used for Clearance Request journey" in {
         val view: Document = createView(formClearance, DeclarationType.CLEARANCE)
-        view.getElementById("submit").text() mustBe messages(saveAndContinueCaption)
+        view.getElementById("submit").text() mustBe messages(continueCaption)
       }
     }
```
Comment (MaciejRewera) [inline test/views/declaration/additionaldeclarationtype/DeclarationTypeViewSpec.scala:144]: This bunch of test cases seems like a good place to use `onEveryDeclarationJourney` method

File: test/views/declaration/additionaldeclarationtype/DeclarationTypeViewSpec.scala [hunk 3]
```diff
@@ -115,69 +115,64 @@ class DeclarationTypeViewSpec extends UnitViewSpec with ExportsTestData with Com
...
 
-    "display 'Save and return' button" when {
+    "not display 'Save and return' button" when {
```
Comment (MaciejRewera) [inline test/views/declaration/additionaldeclarationtype/DeclarationTypeViewSpec.scala:146]: Same here

File: test/views/declaration/additionaldeclarationtype/DeclarationTypeViewSpec.scala (hunk 2 above)
Comment (SteveSugden) [reply test/views/declaration/additionaldeclarationtype/DeclarationTypeViewSpec.scala:144]: I thought about that.  For about 200mS until I realised that the views are created from journey-specific components.   So I just fix what I broke.

**Pull request merged**